.github/workflows/lint.yml

```
@@ -124,15 +130,15 @@ jobs:
         uses: actions/checkout@v2
       - name: Install markdown-toc
         run: npm install -g markdown-toc
-      - name: Regenerate ToCs
+      - name: Regenerate ToCs and check that they didn't change
         run: |
           set -eux
           export PATH=~/.npm-global/bin:"$PATH"
           for FILE in $(git grep -Il '<!-- toc -->' -- '**.md'); do
             markdown-toc --bullets='-' -i "$FILE"
           done
-      - name: Assert that regenerating the ToCs didn't change them
-        run: .github/scripts/report_git_status.sh
+
+          bash .github/scripts/report_git_status.sh
```

Why the explicit `bash` here?
```suggestion
          .github/scripts/report_git_status.sh
```

---

Also, if you're planning to use this directly for `make lint`, I'm pretty sure this will fail anytime the developer has any uncommitted changes, which I don't think is what you want.

---

I wasn't planning on it, it's nice to have the shortcut but it depends on `npm` to install `markdown-toc`, which I don't think we should require to run the rest of the linters (they only depend on Python/`pip` which we can probably assume is fine)